src/components/TradeWidget/TxMessage.tsx

```diff
@@ -194,14 +220,34 @@ export const TxMessage: React.FC<TxMessageProps> = ({ sellToken, receiveToken, n
   const displaySellToken = displayTokenSymbolOrLink(sellToken)
   const displayReceiveToken = displayTokenSymbolOrLink(receiveToken)
 
-  const { isLoading, isLowVolume } = useLowVolumeAmount({ sellToken, networkId, sellTokenAmount })
+  const {
+    isLoading,
+    isLowVolume,
+    roundedUpAmount: recommendedAmount,
+    roundedUpAmountInOwl: roundedAmountInUSD,
+  } = useLowVolumeAmount({
+    sellToken,
+    networkId,
+    sellTokenAmount,
+  })
+
+  const adjustedRecommendedAmount = useMemo(
+    () => (recommendedAmount ? adjustPrecision(recommendedAmount?.toString(), 2) : ''),
+    [recommendedAmount],
+  )
+
+  const setRecommendedSellAmount = (): void => {
+    if (!adjustedRecommendedAmount) return
+    setValue('sellToken', adjustedRecommendedAmount, true)
```

Comment:
i think this may be fixed in v6 with `setValue('sellToken', adjustedRecommendedAmount, { shouldDirty: true })` but im not sure